### PR TITLE
Replace `require` with `import` for error section images

### LINKED ISSUE #2607: Replace `require` with `import` for error section images
## Problem

In the error section, we use `require` for images. `require` will be deprecated in Nuxt 3.

## Description

We should move the `error_images` folder (images only, not the `json` file) to `/static` folder, and use the static URLs.

https://github.com/WordPress/openverse/blob/e404f51269e2291726672785ed0a1604994a2eaf/frontend/src/components/VErrorSection/VErrorImage.vue#L54

Change the code above to 
```
src: `/error_images/${image.file}.jpg`, 
```

The `json` file should be renamed and moved up, from `assets/error_images/image_info.json` to `assets/error_images.json`.

## Additional details
This issue is very similar to #2601

### PR DESCRIPTION
**Pull request by viktoriussuwandi**
## Fixes

Fixes #2607 by @obulat 

## Description

Hi [obulat](https://github.com/obulat),
just trying to help on this issue, please check it out

[Remove src:error_images on frontend/src/components/VErrorSection/VErr](https://github.com/WordPress/openverse/commit/e045d94f627477fe93cf90833f8a4b0d1006fa04)

[Rename frontend/src/assets/error_images/image_info.json to frontend/s…](https://github.com/WordPress/openverse/commit/670c94b539755fc1127d49ed705f7a612c1e2cbd)

## Checklist

- [X] My pull request has a descriptive title (not a vague title like`Update index.md`).
- [X] My pull request targets the _default_ branch of the repository (`main`) or a parent feature branch.
- [X] My commit messages follow [best practices][best_practices].
- [X] My code follows the established code style of the repository.
- [X] I added or updated tests for the changes I made (if applicable).
- [X] I added or updated documentation (if applicable).
- [X] I tried running the project locally and verified that there are no visible errors.
- [X] I ran the DAG documentation generator (if applicable).

[best_practices]:
  https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

File: frontend/src/components/VErrorSection/VErrorImage.vue [hunk 1]
```diff
@@ -51,7 +51,7 @@ export default defineComponent({
         const errorImage: ErrorImage = {
           ...image,
           originalTitle: image.title,
-          src: require(`~/assets/error_images/${image.file}.jpg`),
+          src: require(`/error_images/${image.file}.jpg`),
```
Comment (obulat) [inline frontend/src/components/VErrorSection/VErrorImage.vue:54]:
```suggestion
          src: `/error_images/${image.file}.jpg`,
```

**Review by obulat — CHANGES_REQUESTED** [review]
This is a great start!

I am sorry that the instructions in the issue were not fully correct. I've added the suggestion to remove the `require`. The `json` import also needs to be updated to `import `imageInfo` from "~/assets/error_images.json"`.
In addition, the images from `assets/error_images` need to be moved to `/static` folder

**Comment by obulat** [comment]
`VErrorImage` line 22 needs to be changed from `import imageInfo from "~/assets/error_images/image_info.json"` to `import imageInfo from "~/assets/error_images.json".

The image files should be moved:
`frontend/src/assets/error_images/depressed_musician.jpg` -> `frontend/src/static/error_images/depressed_musician.jpg`
`frontend/src/assets/error_images/waiting_for_a_bite.jpg` -> `frontend/src/static/error_images/waiting_for_a_bite.jpg`

File: frontend/assets/image_info.json
Comment (obulat) [inline frontend/assets/image_info.json:1]: This file should be renamed to `error_images` (so it would be `frontend/src/assets/error_images.json`)

**Review by obulat — APPROVED** [review]
All problems fixed ✨ 
Thank you for your contribution, @viktoriussuwandi , and welcome to Openverse 😄

**Review by dhruvkb — APPROVED** [review]
Verified working in Storybook, thanks for the fix @viktoriussuwandi!

**Pull request merged**